### ISSUE TITLE
acme.json always at root

### ISSUE THREAD

**Issue by trajano** [containous/traefik#5360]
### Do you want to request a *feature* or report a *bug*?

Bug

### Did you try using a 1.7.x configuration for the version 2.0?

- [ ] Yes
- [X] No

### What did you do?
I got LetsEncrypt working now, but I noticed something was off in that when I restart the server it appears to recreate the certificate again.  The commands I am using to configure traefik are:

      - --certificatesresolvers.default.acme.httpchallenge.entrypoint=http
      - --certificatesResolvers.default.acme.email=support@trajano.net
      - --certificatesResolvers.default.acme.storage=/letsencrypt/acme.json

I have this volumes block on the service

    volumes:
      - letsencrypt:/letsencrypt

To correspond to the following in the top level compose file:
```
volumes:
  letsencrypt:
```

I see the volume when I do `docker volume ls` but when I try to view it's contents it is empty using this command

    docker run -it -v edge_letsencrypt:/mnt bash ls -la //mnt

### What did you expect to see?

`acme.json` present

### What did you see instead?

`/mnt` was empty

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
$ docker run traefik:v2.0.0-rc3 version
Version:      2.0.0-rc3
Codename:     montdor
Go version:   go1.13
Built:        2019-09-10T17:10:04Z
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
# (paste your configuration here)
```

### If applicable, please paste the log output in DEBUG level (`--log.level=DEBUG` switch)

```
(paste your output here)
```

I also tried to see what's inside the `overlay2` folders to peek into the details

```
/host/var/lib/docker/overlay2 # find . -name "acme.json"
./4f82ac74bd8b3d0fe4de3aafb5a54cc93707219a0677edf22df0180be2d579c7/diff/acme.json
./4f82ac74bd8b3d0fe4de3aafb5a54cc93707219a0677edf22df0180be2d579c7/merged/acme.json
/host/var/lib/docker/overlay2 # 
```
The files are there and are not empty (they actually contain the proper data)

**Comment by trajano** [comment]
https://community.containo.us/t/letsencrypt-json-file-is-not-being-stored/1470/9

**Comment by trajano** [comment]
Issue is still persisting after the 2,0.0 release

**Comment by trajano** [comment]
The problem is in the CLI configuration, environment variable configuration works
```
  TRAEFIK_CERTIFICATESRESOLVERS_DEFAULT_ACME_EMAIL: support@trajano.net
  TRAEFIK_CERTIFICATESRESOLVERS_DEFAULT_ACME_HTTPCHALLENGE_ENTRYPOINT: http
  TRAEFIK_CERTIFICATESRESOLVERS_DEFAULT_ACME_STORAGE: /etc/traefik/acme/default.json
```

**Comment by ldez** [comment]
It's just a problem related to case:

```
  - --certificatesresolvers.default.acme.httpchallenge.entrypoint=http
  - --certificatesResolvers.default.acme.email=support@trajano.net
  - --certificatesResolvers.default.acme.storage=/letsencrypt/acme.json
```

`certificatesresolvers` vs `certificatesResolvers` => you need to use the same case 

```
  - --certificatesresolvers.default.acme.httpchallenge.entrypoint=http
  - --certificatesresolvers.default.acme.email=support@trajano.net
  - --certificatesresolvers.default.acme.storage=/letsencrypt/acme.json
```
or 

```
  - --certificatesResolvers.default.acme.httpchallenge.entrypoint=http
  - --certificatesResolvers.default.acme.email=support@trajano.net
  - --certificatesResolvers.default.acme.storage=/letsencrypt/acme.json
```

**Comment by trajano** [comment]
it would be nice to have an error message that says it or at least make sure the documentation pages were consistent on this one.  II think I copied from two different pages and eventually got the issue.

**Comment by ldez** [comment]
In the next path release, the problem will be fixed: https://github.com/containous/traefik/pull/5438

The case will no longer need to be consistent.

**Comment by traefiker** [comment]
Closed by #5438.
